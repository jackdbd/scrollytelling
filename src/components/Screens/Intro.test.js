import React from "react";
import { shallow, mount, render } from "enzyme";
import "../../setupEnzymeTests";
import Intro from "./Intro";

describe("<Intro />", () => {
  it("has two <p> elements", () => {
    const wrapper = mount(<Intro />);
    expect(wrapper.find("p")).toHaveLength(2);
  });
  it("contains 'Scroll to start'", () => {
    const wrapper = mount(<Intro />);
    expect(
      wrapper
        .find("p")
        .at(1)
        .text()
    ).toContain("Scroll to start");
  });
  it("matches the snapshot", () => {
    expect(<Intro />).toMatchSnapshot();
  });
});
