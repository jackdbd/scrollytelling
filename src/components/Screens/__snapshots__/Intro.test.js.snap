// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Intro /> matches the snapshot 1`] = `
<Intro
  backgroundColor="#d3d3d3"
  fontFamily="Arial"
  title="Intro title"
/>
`;
